release_tester/arangodb/starter/manager.py

```diff
@@ -1280,6 +1284,15 @@ def count_occurances_in_starter_log(self, substring: str):
         number_of_occurances = self.get_log_file().count(substring)
         return number_of_occurances
 
+    def stop_dbserver(self):
```

Comment:
this should not be a method specific to db-servers. 
It should rather be
stop_instances(instance_type)